unittest/gunit/fake_mdl.cc

```diff
@@ -0,0 +1,33 @@
+/* Copyright (c) 2019, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Shouldn't this be Percona? Or facebook? Not sure which, but I don't think we should add oracle copyright to files added by us.

Reply:
updated